src/util/list.h

```diff
@@ -122,17 +153,17 @@ void remove_node(list_t *list, int key);
 /**
 @brief Returns list head node.
 */
-static inline list_node_t *first_list(list_t *list) { return head_list(list, false); }
```

Comment:
This is valid C.  I would like to think that inlining makes these slightly faster.

Reply:
The compiler will inline expressions like these. "inline" is not suported in older versions of C.